rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/NodeSource.java

```diff
@@ -673,6 +673,15 @@ public void acquireNode() {
         infrastructureManager.acquireNode();
     }
 
+    public void acquireNodes(int n, long timeout, Map<String, ?> nodeConfiguration) {
+        if (toShutdown) {
+            logger.warn("[" + name + "] acquireNodes request discarded because node source is shutting down");
+            return;
+        }
+
+        infrastructureManager.acquireNodes(n, timeout, nodeConfiguration);
+    }
+
     public void acquireNodes(int n, Map<String, ?> nodeConfiguration) {
```

Comment:
Why not doing the changing this method's signature? I guess both are needed right?

Reply:
Yes. This method with old signature is used by Dynamic policy and SimpleJMXNodeSourcePolicy. So I prefer to keep it, instead of modifying all its dependencies.

Reply:
Fair enough.